### PR TITLE
[FLOC 4319] Add Open vStorage backend driver to documentation

### PR DESCRIPTION
**Pull request by davidoliverSP2**
Fixes 4319

This adds documentation for using the Open vStorage backend.

### CODE REVIEW

**Comment by davidoliverSP2** [comment]
Link to the Jenkins build: http://clusterhq-staging-docs.s3-website-us-east-1.amazonaws.com/openvstorage-backenddriver-FLOC-4319/

File: docs/flocker-features/open-vstorage-configuration.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+==================================
+Open vStorage Block Device Backend
+==================================
+
+.. begin-body
+
+Open vStorage provides a plugin for Flocker integration allowing Dockerized applications to use a vPool on Open vStorage.
+
+For more information, visit the following GitHub repository: `Open vStorage Flocker driver on GitHub`_
```
Comment (zubron) [inline docs/flocker-features/open-vstorage-configuration.rst]: ❓ Should this line have a full stop at the end?

File: docs/flocker-features/open-vstorage-configuration.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+To start using the Open vStorage Flocker plugin, see the `Open vStorage documentation`_.
+
+.. XXX FLOC 2443 to expand this Backend storage section
```
Comment (zubron) [inline docs/flocker-features/open-vstorage-configuration.rst]: Is this comment needed? FLOC-2443 is complete.

**Comment by zubron** [comment]
Looks good - thanks @davidoliverSP2! Is there a reason why we include two copies of the logo?

Please address comments and merge :)

**Comment by sarum90** [comment]
Lint and Sphinx passed, merging presently.

**Pull request merged**